webapp/components/Comment/Comment.vue

```diff
@@ -141,6 +151,10 @@ export default {
     },
   },
   methods: {
+    reply() {
+      const message = { slug: this.comment.author.slug, id: this.comment.author.id }
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-16T10:48:35Z" title="Thursday, January 16th 2020, 11:48:35 am +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -141,6 +148,10 @@ export default {
     },
   },
   methods: {
+    reply(comment) {
+      const message = { slug: this.comment.author.slug, id: this.comment.author.id }
```


You could also just send the `author` as a message (contains `id` and `slug`).
```suggestion
      const { author } = this.comment
      this.$emit('reply', author)
```